corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -46,11 +53,6 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 appId = this.model.collection.appId,
                 value = this.options.model.get('value');
 
-            // Initial values are sent from formplayer as strings, but dropdowns expect an integer
-            if (value && this.options.model.get('input') === "select1") {
-                value = parseInt(value);
```

Comment:
is this not still needed for `select1`?

Reply:
I have moved type checking to the template [here](https://github.com/dimagi/commcare-hq/pull/29808/files#diff-5b5306df02c3e60efa4688371a6f46de4e1f3733d92b9bf0b3e6f17917ace93cR42), so it's not needed here.